dev/checkstyle.xml

```diff
@@ -64,6 +64,11 @@
         <property name="message" value="No trailing whitespace allowed."/>
     </module>
 
+    <module name="RegexpSingleline">
+        <property name="format" value="throw new OutOfMemoryError"/>
```

Comment:
Per my suggestion below, how about the regex `throw new \\w+Error\\(`

Reply:
Thanks, done in 6e49bb8.